tests/python/gaia-ui-tests/gaiatest/apps/system/regions/fullscreen_dialog.py

```diff
@@ -0,0 +1,68 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+
+from gaiatest.apps.base import Base
+from gaiatest.apps.base import PageRegion
+from gaiatest.form_controls.binarycontrol import GaiaBinaryControl
+from gaiatest.apps.search.regions.browser import Browser
+
+class FullScreenDialog(PageRegion):
+
+    _fullscreen_dialog_locator = (By.ID, 'fullscreen-dialog-overlay')
+
+    def __init__(self, marionette):
+        Base.__init__(self, marionette)
+        self.marionette.switch_to_frame()
+        self.root_element = Wait(self.marionette).until(
+            expected.element_present(*self._fullscreen_dialog_locator))
+
+    @property
+    def is_empty(self):
+        return self.marionette.execute_script("""
```

Comment:
No, the tracking protection dialog is removed from the dom after tapping the "Got it" button.
That means self.root_element is not there anymore.
I guess I could use `System(self.marionette).is_element_present(*self._tracking_notice_locator)` and then I could get rid of this rather weird FullScreenDialog class.